source/frontend/src/features/properties/map/acquisition/common/GenerateForm/hooks/useGenerateH120.ts

```diff
@@ -44,20 +51,30 @@ export const useGenerateH120 = () => {
       true,
     );
     const h120CategoriesPromise = getH120Categories.execute();
+    const interestHoldersPromise = getAcquisitionInterestHolders.execute(
+      compensation.acquisitionFileId,
+    );
+    const payeePromise = getCompensationRequisitionPayee.execute(compensation.id);
```

Comment:
hmm - something doesn't add up here. Why would we call the backend "get payee by ID" passing the compensation ID...

did you mean `compensation.payees[0].id` ?

Reply:
yes, will fix.